en/00_Getting_Started/00_Server_Requirements.md

```diff
@@ -90,13 +90,6 @@ also needs write access for the webserver user to the following locations:
   [Manifests](/developer_guides/execution_pipeline/manifests), [Object Caching](/developer_guides/performance/caching)
   and [Partial Template Caching](/developer_guides/templates/partial_template_caching).
   See [Environment Management](/getting_started/environment_management).
-- `.graphql-generated`: silverstripe/graphql uses this directory. This is where your schema is
-  stored once it [has been built](/developer_guides/graphql/getting_started/building_the_schema). Best practice
-  is to create it ahead of time, but if the directory doesn't exist and your project root is writable, the GraphQL
-  module will create it for you.
-- `public/_graphql`: silverstripe/graphql uses this directory. It's used for
-  [schema introspection](/developer_guides/graphql/tips_and_tricks#schema-introspection). You should treat this folder
-  the same way you treat the `.graphql-generated` folder.
```

Comment:
For future reference note this was moved to https://github.com/silverstripe/silverstripe-graphql/pull/596/files#diff-ca13d5c12f2452e25ca5d738ffff7ae3f225effced44ad26f5999c3c16b91fdf